tests/test_hear_keyboard_interrupt.py

```
@@ -58,37 +66,84 @@
+		This test:
+			1. Starts a multicast server (with coverage tracking)
+			2. Waits for server initialization
+			3. Sends a SIGINT signal to simulate Ctrl+C
+			4. Verifies that the server exits with the expected status code
+
+		Success criteria:
+			- Server must exit with status code 130 (standard SIGINT exit code)
+			- Server must not exit with status code 2 (invalid arguments)
+		"""
+		theResult: bool = False
+		fail_fixture: str = str("C^INT --> HEAR == error")
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)